pkg/mdformatter/mdformatter.go

```diff
@@ -109,11 +110,31 @@ func (FormatFrontMatter) TransformFrontMatter(_ SourceContext, frontMatter map[s
 	}
 	sort.Sort(sort.Reverse(sort.StringSlice(keys)))
 
-	b := bytes.NewBuffer([]byte("---\n"))
+	b := bytes.NewBuffer([]byte("---"))
 	for _, k := range keys {
-		_, _ = fmt.Fprintf(b, "%v: %v\n", k, frontMatter[k])
+		// check if frontMatter[k] is a map
```

Comment:
Let's use sentences in comments (Upper case first letter and trailing periods) 🤗

Reply:
Sure